kuzminskiy_andrey/gen_graph.cpp

```diff
@@ -0,0 +1,88 @@
+// uncomment the 85nd line to output the graph
+
+#include <iostream>
+#include <vector>
+
+class Graph {
+  using VertexId = int;
+  using EdgeId = int;
+
+ public:
+  void add_vertex() { kVerticesCount++; }
+
+  void add_edge(VertexId from_vertex_id, VertexId to_vertex_id) {
+    edges.emplace(edges.end(),
+                  (Edge(kEdgesCount++, from_vertex_id, to_vertex_id)));
```

Comment:
1) Лучше просто `emplace_back` использовать
```suggestion
    edges.emplace_back(kEdgesCount++, from_vertex_id, to_vertex_id);
```
2) Лучше выделить логику по генерации нового уникального id в отдельную функцию, например
```cpp
VertexId get_new_edge_id() {
  return kEdgesCount++;
}
```